kafka_consumer/datadog_checks/kafka_consumer/new_kafka_consumer.py

```diff
@@ -192,17 +194,6 @@ def _get_highwater_offsets(self):
             # Loop until all futures resolved.
             self.kafka_client._wait_for_futures(highwater_futures)
 
-    # FIXME: This is using a workaround to skip socket wakeup, which causes blocking
```

Comment:
To reviewer: To make it easier to see which functions were originally from the `kafka-python` library, I am moving these functions to the bottom.